packages/vuetify/src/components/VIcon/VIcon.sass

```diff
@@ -6,9 +6,6 @@
   color: map-deep-get($material, 'text-color')
   opacity: map-deep-get($material, 'icons', 'active')
 
-  &:focus
-    opacity: map-deep-get($material, 'icons', 'focused')
```

Comment:
![image](https://media1.tenor.com/images/c77112065f6515ebd672aa64c9439cc4/tenor.gif)

Reply:
Also v-rating now has a double ripple
![image](https://user-images.githubusercontent.com/16421948/73750896-247acd80-47b2-11ea-861b-4e0b67bcb1b7.png)

Reply:
Might be able to remove the need for v-btn in that component, looking into it.

Reply:
Also there was an overwhelming majority for not having both.